From ebbf6f5a8991239597c74274d47f10d3c8060b24 Mon Sep 17 00:00:00 2001
From: Preston Petrie <preston159@users.noreply.github.com>
Date: Mon, 2 Jun 2025 19:29:59 +0200
Subject: [PATCH 6/7] un-remove software

---
 Containerfile | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/Containerfile b/Containerfile
index 2af95695..4f7d465e 100644
--- a/Containerfile
+++ b/Containerfile
@@ -192,7 +192,7 @@ RUN --mount=type=cache,dst=/var/cache \
         firefox \
         firefox-langpacks \
         toolbox \
-        htop && \
+        && \
     /ctx/cleanup
 
 # Install new packages
@@ -381,9 +381,7 @@ RUN --mount=type=cache,dst=/var/cache \
             plasma-welcome-fedora \
             plasma-discover-kns \
             kcharselect \
-            kde-partitionmanager \
-            plasma-discover \
-            konsole && \
+            && \
         dnf5 -y install --allowerasing \
             https://kojipkgs.fedoraproject.org/packages/kwin/6.5.1/2.fc43/x86_64/kwin-6.5.1-2.fc43.x86_64.rpm \
             https://kojipkgs.fedoraproject.org/packages/kwin/6.5.1/2.fc43/x86_64/kwin-common-6.5.1-2.fc43.x86_64.rpm \
-- 
2.51.1

