From defc20071a4d30e3be5a96d4203a0ad542a8cb29 Mon Sep 17 00:00:00 2001
From: Preston Petrie <preston159@users.noreply.github.com>
Date: Mon, 2 Jun 2025 19:29:59 +0200
Subject: [PATCH 6/8] un-remove software

---
 Containerfile | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/Containerfile b/Containerfile
index a4af2415..adf6065f 100644
--- a/Containerfile
+++ b/Containerfile
@@ -231,8 +231,7 @@ RUN --mount=type=cache,dst=/var/cache \
         ublue-os-update-services \
         firefox \
         firefox-langpacks \
-        htop && \
-    	dnf5 -y swap kde-partitionmanager gnome-disk-utility && \
+        && \
     /ctx/cleanup
 
 # Install new packages
@@ -440,7 +439,7 @@ RUN --mount=type=cache,dst=/var/cache \
             plasma-welcome-fedora \
             plasma-browser-integration \
             kcharselect \
-            konsole && \
+            && \
         sed -i '/<entry name="launchers" type="StringList">/,/<\/entry>/ s/<default>[^<]*<\/default>/<default>preferred:\/\/browser,applications:steam.desktop,applications:net.lutris.Lutris.desktop,applications:org.kde.discover.desktop,preferred:\/\/filemanager<\/default>/' /usr/share/plasma/plasmoids/org.kde.plasma.taskmanager/contents/config/main.xml && \
         sed -i '/<entry name="favorites" type="StringList">/,/<\/entry>/ s/<default>[^<]*<\/default>/<default>preferred:\/\/browser,steam.desktop,net.lutris.Lutris.desktop,systemsettings.desktop,org.kde.dolphin.desktop,org.kde.kate.desktop,org.kde.discover.desktop,system-update.desktop<\/default>/' /usr/share/plasma/plasmoids/org.kde.plasma.kickoff/contents/config/main.xml && \
         sed -i 's/^Exec=plasma-discover/& --backends flatpak-backend/' /usr/share/applications/org.kde.discover.desktop && \
-- 
2.49.0

