From f7cbf92cc501174cdd5d6b0a2027160fcbaaa324 Mon Sep 17 00:00:00 2001
From: Preston Petrie <preston159@users.noreply.github.com>
Date: Mon, 2 Jun 2025 19:30:14 +0200
Subject: [PATCH 7/7] add software

---
 Containerfile | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/Containerfile b/Containerfile
index 3358a338..b14a1cc6 100644
--- a/Containerfile
+++ b/Containerfile
@@ -355,6 +355,15 @@ RUN --mount=type=cache,dst=/var/cache \
     chmod +x /usr/bin/ujust-picker && \
     /ctx/cleanup
 
+# my packages
+RUN --mount=type=cache,dst=/var/cache \
+    --mount=type=cache,dst=/var/log \
+    --mount=type=bind,from=ctx,source=/,target=/ctx \
+    --mount=type=tmpfs,dst=/tmp \
+    mkdir -p /var/lib/alternatives && \
+    /ctx/build.sh && \
+    /ctx/cleanup
+
 # Configure KDE & GNOME
 RUN --mount=type=cache,dst=/var/cache \
     --mount=type=cache,dst=/var/log \
-- 
2.51.0

